queue/lib/txgh-queue.rb

```diff
@@ -0,0 +1,8 @@
+module TxghQueue
+  autoload :Backends,      'txgh-queue/backends'
```

Comment:
now i remember the other thing activesupport would have helped with ... autoloading by convention